Eat_and_Greet/src/style.tsx

```diff
@@ -14,12 +16,60 @@ const style = StyleSheet.create({
         fontSize:36,
         fontWeight:'bold'
     },
+    HeaderText: {
```

Comment:
Is the idea that these are the text's fonts and margins we will use everywhere? Split these into minimal parts (Font family, font size, fontWeight) and put the other parts in local styling so we can reuse more.